pkg/proxy/kafka.go

```diff
@@ -31,6 +31,7 @@ import (
 
 	"github.com/optiopay/kafka/proto"
 	"github.com/sirupsen/logrus"
+	"io"
```

Comment:
Move `"io"` and `"net"` into the stdlib group of imports above.

Reply:
will do.

Reply:
done